tools/flang1/flang1exe/accpp.c

```diff
@@ -913,6 +927,22 @@ accpp(void)
       delete(*cp); /* should check if IDENT */
     }
   }
+  if (g_list_macros) {
+    acc_display_macros();
+    
+    FREE(deftab);
+    FREE(hashrec);
+    /* FREE(inclstack); Do not free this, gbl.curr_file points to it */
+    FREE(_ifs);
+    FREE(linebuf);
+
+    if (incllist != 0) {
```

Comment:
please do not compare pointers with bare integers... `if (incllist != NULL)` or just `if (!incllist)` would do.